packages/components/src/RecurringSelect/DateCellCheckbox.module.css

```diff
@@ -54,7 +54,7 @@
 
 .dayCheckboxInput:disabled + .dayCheckbox {
   color: var(--color-disabled);
-  background: var(--color-interactive--background);
+  background: var(--color-disabled--background);
```

Comment:
Sorry I think this is my bad on the `suggestion` comment - this should be `--color-disabled--secondary`!

Reply:
noticed I did the same mistake in sg-1, thanks!